tests/flow/test_index_updates.py

```diff
@@ -2,6 +2,8 @@
 import sys
 import random
 import string
+import utils.multiproc as mlp
+from utils.multiproc import *
```

Comment:
```python
import utils.multiproc as mlp
from mlp import *
```

Reply:
This doesn't wotk:

Traceback (most recent call last):
  File "/usr/local/bin/RLTest", line 8, in <module>
    sys.exit(main())
  File "/usr/local/lib/python3.9/site-packages/RLTest/__main__.py", line 673, in main
    RLTest().execute()
  File "/usr/local/lib/python3.9/site-packages/RLTest/__main__.py", line 412, in __init__
    self.loader.load_spec(self.args.test)
  File "/usr/local/lib/python3.9/site-packages/RLTest/loader.py", line 80, in load_spec
    self.load_files(dirname, module_name)
  File "/usr/local/lib/python3.9/site-packages/RLTest/loader.py", line 85, in load_files
    module = imp.load_module(module_name, module_file, filename,
  File "/usr/local/Cellar/python@3.9/3.9.5/Frameworks/Python.framework/Versions/3.9/lib/python3.9/imp.py", line 234, in load_module
    return load_source(name, filename, file)
  File "/usr/local/Cellar/python@3.9/3.9.5/Frameworks/Python.framework/Versions/3.9/lib/python3.9/imp.py", line 171, in load_source
    module = _load(spec)
  File "<frozen importlib._bootstrap>", line 711, in _load
  File "<frozen importlib._bootstrap>", line 680, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 855, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/Users/ofirmoskovich/RedisGraph/tests/flow/test_index_updates.py", line 6, in <module>
    from mlp import *
ModuleNotFoundError: No module named 'mlp'